File: aifupan_project/ReviewAnalysis/bo/video/UpdateVideoSizeDurationBo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ReviewAnalysis.bo.video
{
    public class UpdateVideoSizeDurationBo
    {
        /// <summary>
        /// 视频id
        /// </summary>
        public string videoId { get; set; }
        /// <summary>
        /// 时长，秒
        /// </summary>
        public long? duration { get; set; }
        /// <summary>
        /// 文件大小，B
        /// </summary>
        public long? fileSize { get; set; }
        /// <summary>
        /// 录制结束时间
        /// </summary>
        public string endTime { get; set; }
        /// <summary>
        /// 是否在录制 0：否 1：是
        /// </summary>
        public int? isRecording { get; set; }
    }
}

File: aifupan_project/fupan-server/replay-words/src/main/java/com/jiuyu/replay/words/bo/video/UpdateVideoSizeDurationBo.java
package com.jiuyu.replay.words.bo.video;

import io.swagger.v3.oas.annotations.media.Schema;
import lombok.Data;

@Data
@Schema(description = "修改视频的大小时长参数")
public class UpdateVideoSizeDurationBo {

    /**
     * 视频id
     */
    @Schema(description = "视频id")
    public String videoId;
    /**
     * 时长，秒
     */
    @Schema(description = "时长，秒")
    public Long duration;
    /**
     * 文件大小，B
     */
    @Schema(description = "文件大小，B")
    public Long fileSize;
    /**
     * 录制结束时间
     */
    @Schema(description = "录制结束时间")
    public String endTime;
    /**
     * 是否在录制 0：否 1：是
     */
    @Schema(description = "是否在录制 0：否 1：是")
    public Integer isRecording;
    /**
     * 用户id
     */
    @Schema(description = "用户id")
    private Long userId;
    /**
     * 租户id
     */
    @Schema(description = "租户id")
    private Long tenantId;
}
